docs/docsite/rst/roadmap/COLLECTIONS_6.rst

```diff
@@ -14,25 +14,25 @@ Release schedule
-:TBD: ansible-core feature freeze.
-:TBD: Start of ansible-core 2.13 betas (weekly, as needed).
-:TBD: Ansible-6.0.0 alpha1 (roughly biweekly ``ansible`` alphas timed to coincide with ``ansible-core-2.13`` pre-releases).
-:TBD: Last day for new collections to be submitted for inclusion in Ansible-6. Collections MUST be reviewed and approved before being included. There is no guarantee that we will review every collection. The earlier your collection is submitted, the more likely it will be that your collection will be reviewed and the necessary feedback can be addressed in time for inclusion.
-:TBD: Community Meeting topic: List any new collection reviews which block release. List any backwards incompatible collection releases that beta1 should try to accommodate.
-:TBD: First ansible-core release candidate, stable-2.13 branch created.
-:TBD: Ansible-6.0.0 alpha2.
-:TBD: Last day for new collections to be **reviewed and approved** for inclusion in Ansible-6.
-:TBD: Community Meeting topic: Decide what contingencies to activate for any blockers that do not meet the deadline.
-:TBD: Ansible-6.0.0 alpha3.
-:TBD: Ansible-core-2.13 released.
-:TBD: Last day for collections to make backwards incompatible releases that will be accepted into Ansible-6.
-:TBD: Create the ansible-build-data directory and files for Ansible-7. New collection approvals will target this.
-:TBD: Ansible-6.0.0 beta1 -- feature freeze [1]_ (weekly beta releases; collection owners and interested users should test for bugs).
-:TBD: Ansible-6.0.0 beta2.
-:TBD: Ansible-6.0.0 rc1 [2]_ [3]_ (weekly release candidates as needed; test and alert us to any blocker bugs).  Blocker bugs will slip release.
-:TBD: Ansible-6.0.0 release.
-:TBD: Release of Ansible-6.1.0 (bugfix + compatible features: every three weeks.)
+:2021-11-30: New Collections can be reviewed for inclusion in Ansible 6. Submit a request to include a new collection in this `GitHub Discussion <https://github.com/ansible-collections/ansible-inclusion/discussions/new>`_.
+:2022-03-28: ansible-core feature freeze.
+:2022-04-11: Start of ansible-core 2.13 betas (biweekly, as needed).
+:2022-04-12: Ansible-6.0.0 alpha1 (roughly biweekly ``ansible`` alphas timed to coincide with ``ansible-core-2.13`` pre-releases).
+:2022-04-26: Last day for new collections to be submitted for inclusion in Ansible-6. Collections MUST be reviewed and approved before being included. There is no guarantee that we will review every collection. The earlier your collection is submitted, the more likely it will be that your collection will be reviewed and the necessary feedback can be addressed in time for inclusion.
+:2022-04-27: Community Meeting topic: List any new collection reviews which block release. List any backwards incompatible collection releases that beta1 should try to accommodate.
+:2022-05-02: First ansible-core release candidate, stable-2.13 branch created.
```

Comment:
core is pulling the branch earlier now. see - https://docs.ansible.com/ansible-core/devel/roadmap/ROADMAP_2_13.html